libr/parse/p/parse_mips_pseudo.c

```diff
@@ -11,12 +11,11 @@
 #include <r_parse.h>
 
 static int can_replace(const char *str, int idx, int max_operands) {
-	int ret = true;
-	if (str[idx] > '9' || str[idx] < '1') ret = false;
+	if (str[idx] > '9' || str[idx] < '1') return false;
 	if (str[idx + 1] != '\x00' && str[idx + 1] <= '9' && str[idx + 1] >= '1')
-		ret = false;
-	if ((int)((int)str[idx] - 0x30) > max_operands) ret = false;
-	return ret;
+		return false;
+	if ((int)((int)str[idx] - 0x30) > max_operands) return false;
```

Comment:
add {} braces even for oneliner conditionals. (yep i know it was not like this before), but its a rule we try to follow from now on :P